tests/behaviour/context.py

```diff
@@ -51,12 +51,15 @@ def __init__(self):
         self.sessions_to_transactions: Dict[TypeDBSession, List[TypeDBTransaction]] = {}
         self.sessions_parallel: List[Future[TypeDBSession]] = []
         self.sessions_parallel_to_transactions_parallel: Dict[Future[TypeDBSession], List[TypeDBTransaction]] = {}
+        self.session_options: Optional[TypeDBOptions] = None
+        self.transaction_options: Optional[TypeDBOptions] = None
```

Comment:
these have to be set to None and _RE SET_ before each test, otherwise cucumber somehow doesn't propagate these values across scenarios